demodoctrine/src/Entity/QuoteLang.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * 2007-2020 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0).
+ * It is also available through the world-wide-web at this URL: https://opensource.org/licenses/AFL-3.0
+ */
+
+namespace PrestaShop\Module\DemoDoctrine\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use PrestaShopBundle\Entity\Lang;
+
+/**
+ * @ORM\Table()
+ * @ORM\Entity()
+ */
+class QuoteLang
+{
+    /**
+     * @var Quote
+     * @ORM\Id
+     * @ORM\ManyToOne(targetEntity="PrestaShop\Module\DemoDoctrine\Entity\Quote", inversedBy="quoteLangs")
+     * @ORM\JoinColumn(name="id_quote", referencedColumnName="id_quote", nullable=false)
+     */
+    private $quote;
+
+    /**
+     * @var Lang
+     * @ORM\Id
+     * @ORM\ManyToOne(targetEntity="PrestaShopBundle\Entity\Lang")
+     * @ORM\JoinColumn(name="id_lang", referencedColumnName="id_lang", nullable=false, onDelete="CASCADE")
+     */
+    private $lang;
+
+    /**
+     * @var string
+     * @ORM\Column(name="content", type="string", nullable=false)
+     */
+    private $content;
+
+    /**
+     * @return Quote
+     */
+    public function getQuote()
+    {
+        return $this->quote;
+    }
+
+    /**
+     * @param Quote $quote
+     * @return $this
+     */
+    public function setQuote($quote)
+    {
+        $this->quote = $quote;
+
+        return $this;
+    }
+
+    /**
+     * @return Lang
+     */
+    public function getLang()
+    {
+        return $this->lang;
+    }
+
+    /**
+     * @param Lang $lang
+     * @return $this
+     */
+    public function setLang($lang)
```

Comment:
declare argument type

Reply:
I wasn't sure if it's compatible with PHP 5.6, I need to check it But it would be better indeed